lib/origen/model/exporter.rb

```diff
@@ -1,6 +1,17 @@
 module Origen
   module Model
     module Exporter
+      # Export the model
+      #
+      # Options defaults:
+      #   include_pins:       true
+      #   include_registers:  true
+      #   include_sub_blocks: true
+      #   include_timestamp:  true
+      #   file_path:          nil     # default path is Origen.root!/vendor/lib/models/name
```

Comment:
I don't think the `:file_path` option should be included here, this is not intended to be a user API - https://github.com/Origen-SDK/origen/blob/master/lib/origen/model/exporter.rb#L12

Reply:
Or if you think having it there will help developers in future, please try and somehow indicate that it is a private option.

Reply:
I started out only adding documentation for the rm_rb_only option. I included file_path only because it was one of the options. I have no heartburn removing it.

Reply:
Removed it.